Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
+    for X in range(1, s.nrows):

```

Comment:
Capital for variable name `X` contradicts [PEP-8](https://www.python.org/dev/peps/pep-0008/).